channeldb/graph.go

```diff
@@ -693,7 +693,7 @@ func (c *ChannelGraph) HasChannelEdge(
 
 		return nil
 	}); err != nil {
-		return time.Time{}, time.Time{}, false, false, err
+		return time.Time{}, time.Time{}, exists, isZombie, err
```

Comment:
If it's returning an error anyway, why's this matter?

Reply:
mainly did it to keep existing behavior, but it shouldn't